2021-06-07 18:23:33,842 - __main__ - INFO - Error detected
Traceback (most recent call last):
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Создание_пассажиров\Сreate_passanger.py", line 31, in genlog
    self.create_passanger()
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Создание_пассажиров\Сreate_passanger.py", line 50, in create_passanger
    driver.find_element_by_css_selector("#j_idt60 > div.nano.layout-tabmenu-nav > ul > li:nth-child(7) > a > div").click()
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 598, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#j_idt60 > div.nano.layout-tabmenu-nav > ul > li:nth-child(7) > a > div"}
  (Session info: chrome=90.0.4430.212)

